### PR TITLE
From TODO: Renamed the namespace to 'Pusher'

### PR DESCRIPTION
**Pull request by tanuck**

### CODE REVIEW

**Comment by zimbatm** [comment]
:+1:

**Comment by GrahamCampbell** [comment]
:+1:

File: src/Client.php [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <?php
 
-namespace pusher;
+namespace Pusher;
 
-use pusher\Version;
+use Pusher\Version;
```
Comment (GrahamCampbell) [inline src/Client.php]: This import is not needed.

Comment (zimbatm) [inline src/Client.php]: Is the import automatically loaded on use ?

The Version::VERSION is used in the userAgent() function here: https://github.com/pusher/pusher-http-php/blob/new-lib/src/Client.php#L112

Comment (GrahamCampbell) [inline src/Client.php]: Imports don't work like that in php. They have nothing to do with autoloading at all. :)

Comment (tanuck) [inline src/Client.php]: @zimbatm It is used in userAgent(), however both classes are in the same namespace so it is not strictly needed.

Comment (zimbatm) [inline src/Client.php]: Thanks @tanuck I understand now

**Pull request merged**